### PR TITLE
feat: add tappable payment method support link

### PR DESCRIPTION
**Pull request by dgacevic**

### CODE REVIEW

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/CardSelection/CardSelectionViewController.swift [hunk 1]
```diff
@@ -40,19 +40,24 @@ class CardSelectionViewController: ItemSelectionViewController {
         }
         
         cell.setup { view in
+            let unavailableText = NSMutableAttributedString().preparePaymentMethodUnavailableText()
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Buy/CardSelection/CardSelectionViewController.swift:43]: presenter logic

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyPresenter.swift [hunk 2]
```diff
@@ -89,12 +89,13 @@ final class BuyPresenter: NSObject, Presenter, BuyActionResponses {
             if let paymentCard = actionResponse.card {
                 switch actionResponse.card?.status {
                 case .statusOk:
+                    let unavailableText = NSMutableAttributedString().preparePaymentMethodUnavailableText()
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyPresenter.swift:92]: duplication

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/OrderPreview/Views/CardDetailsView.swift [hunk 3]
```diff
@@ -140,7 +140,6 @@ class CardDetailsView: FEView<CardDetailsConfiguration, CardDetailsViewModel> {
         cardNumberLabel.configure(with: config?.cardNumber)
         expirationLabel.configure(with: config?.expiration)
         moreButton.configure(background: config?.moreButton)
-        errorLabel.configure(with: config?.error)
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Buy/OrderPreview/Views/CardDetailsView.swift:143]: where do we configure it

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/OrderPreview/Views/CardDetailsView.swift [hunk 4]
```diff
@@ -160,6 +159,10 @@ class CardDetailsView: FEView<CardDetailsConfiguration, CardDetailsViewModel> {
         
         errorLabel.setup(with: viewModel?.errorMessage)
         errorLabel.isHidden = viewModel?.errorMessage == nil
+        errorLabel.isUserInteractionEnabled = viewModel?.errorMessage != nil
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Buy/OrderPreview/Views/CardDetailsView.swift:162]: why disable interaction if the view is hidden on 161

Comment (dgacevic) [reply brd-ios/breadwallet/src/Redesign/Scenes/Buy/OrderPreview/Views/CardDetailsView.swift:162]: it's not about disabling the interaction if view is hidden - the interaction is off by default and this code switches it on if the error is present

**Review by kenanmamedoff — APPROVED** [review]

**Pull request merged**